lesson3/ToDoList2/app/src/main/java/erica/todolist2/MainActivity.java

```diff
@@ -42,10 +40,13 @@ public boolean onCreateOptionsMenu(Menu menu) {
         return true;
     }
 
+    /**
+     * When "Settings" is called uses replaceFragment method defined below to replace the current
+     * fragment with settings fragment
+     */
     @Override
-    public boolean onOptionsItemSelected(MenuItem item) {                                           //there is an option called "Settings" in the menu
```

Comment:
I'd put this comment on a new line instead of in-line with the function declaration